packages/sources/package.json

```diff
@@ -28,8 +28,8 @@
         "awesome-debounce-promise": "^2.1.0"
     },
     "devDependencies": {
-        "@data-driven-forms/pf4-component-mapper": "^2.6.7",
-        "@data-driven-forms/react-form-renderer": "^2.6.7",
+        "@data-driven-forms/pf4-component-mapper": "^2.8.3",
```

Comment:
make sure to bump the peer dependency as well (good practices and all that)

Reply:
Oh, you are right :sob: